Detectors/TRD/qc/include/TRDQC/Tracking.h

```diff
@@ -44,6 +44,7 @@ namespace trd
 
 struct TrackQC {
   int type;                          ///< 0 TPC-TRD track; 1 ITS-TPC-TRD track
+  unsigned int refGlobalTrackId;     ///< raw GlobalTrackID of the seeding track (either ITS-TPC or TPC)
```

Comment:
why are you not storing it directly as `o2::dataformats::GlobalTrackID`?

Reply:
No particular reason actually..